.github/workflows/manual_workflow.yaml

```diff
@@ -0,0 +1,73 @@
+name: Publish Specific Charts
+on:
+  workflow_dispatch:
+    inputs:
+      charts:
+        description: 'Charts to publish'
```

Comment:
is this an array or string. Is this applicable for one or multiple charts at a time.